samples/wifi/raw_tx_packet/src/main.c

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+/** @file
+ * @brief Wi-Fi Raw Tx Packet sample
+ */
+
+#include <zephyr/logging/log.h>
+LOG_MODULE_REGISTER(raw_tx_packet, CONFIG_LOG_DEFAULT_LEVEL);
+
+#include <nrfx_clock.h>
+#include <zephyr/kernel.h>
+
+#include "net_private.h"
+
+int try_wifi_connect(void);
+
+int main(void)
+{
+	int status;
+
+	LOG_INF("Starting %s with CPU frequency: %d MHz", CONFIG_BOARD, SystemCoreClock/MHZ(1));
+	k_sleep(K_SECONDS(1));
+
+	status = try_wifi_connect();
+	if (status < 0) {
+		return status;
+	}
```

Comment:
`return try_wifi_connect();`

Reply:
In success case(once connected), we need to continue sending packets. Returning from try_wifi_connect doesn't work that way, right?

Reply:
In this commit we aren't doing anything, anyways, its fine as we change it later.